pandas/core/indexing.py

```diff
@@ -662,7 +662,9 @@ def _ensure_listlike_indexer(self, key, axis=None, value=None):
         if self.ndim != 2:
             return
 
-        if isinstance(key, tuple) and not isinstance(self.obj.index, ABCMultiIndex):
+        if isinstance(key, tuple) and (
+            isinstance(key[0], slice) or not isinstance(self.obj.index, ABCMultiIndex)
+        ):
             # key may be a tuple if we are .loc
             # if index is not a MultiIndex, set key to column part
```

Comment:
Thanks @phofl for the PR. does this comment need updating?

Reply:
Thx